db/migrate/20200826131359_add_missing_foreign_keys_to_models_related_to_articles.rb

```diff
@@ -0,0 +1,6 @@
+class AddMissingForeignKeysToModelsRelatedToArticles < ActiveRecord::Migration[6.0]
+  def change
+    add_foreign_key :buffer_updates, :articles, on_delete: :cascade, validate: false
+    add_foreign_key :rating_votes, :articles, on_delete: :cascade, validate: false
```

Comment:
`articles.rating_votes` goes, it's `user.rating_votes` that stays ;-)

see https://github.com/forem/forem/pull/10011#discussion_r477450050